### PR TITLE
Remove custom windows chrome causing many issues #658

### PR DESCRIPTION
**Pull request by lodle**

### CODE REVIEW

**Comment by karolherbst** [comment]
I like cleaning up, but what are the effects and what will be different?

File: src/include/wx_controls/gcCustomFrame.h [hunk 1]
```diff
@@ -68,154 +66,19 @@ class gcCustomFrame : public T, public FrameIcon
...
-			m_pCustomFrame->destroy();
-			m_pCustomFrame->onActiveEvent -= delegate(&onActiveEvent);
-		}
-#endif
-
-		return T::Destroy();
-	}
-
-	void init(gcCustomFrameImpl* custFrame)
+	virtual ~gcCustomFrame()
```
Comment (karolherbst) [inline src/include/wx_controls/gcCustomFrame.h]: do we still need this destructor? I think it would be safe to remove it fully

**Comment by lodle** [comment]
It will look like a normal windows application instead of having the integrated title bar into the top of the app

**Comment by karolherbst** [comment]
I've tested it on windows 8, it seems to work fine.

**Pull request merged**